INSTALL.md

```diff
@@ -101,8 +102,13 @@ echo "addnode=dnsseed.hush.quebec" >> ~/.hush/hush.conf
 ```
 
 ## Run a HUSH Node
+It can take over 8 hours to fully sync.
```

Comment:
It can actually take days to sync for many people on home ISP internet. Maybe something more specific like "It takes a long time to fully sync. As of May 2018, the Hush blockchain is about 2GB to download, which does not include the proving key"